### PR TITLE
Closes #114 - Add gages to Fire Hydrology - Query by Fire Perimeter workflow

### LINKED ISSUE #114: Add gages to Fire Hydrology - Query by Fire Perimeter workflow
- [x] Grab streamgages from the service response-- must complete https://github.com/USGS-WiM/nldi_polygon_query/issues/5 first
- [x] Display the gages on the map: they already exist on the map, so maybe change symbology? and reset symbology when workflow is complete
- [x] List the gages in the report

### PR DESCRIPTION
**Pull request by amedenblik**
Closes #114

### CODE REVIEW

File: src/app/map/map.component.ts
Comment (harper-wavra) [inline src/app/map/map.component.ts:628]: There seem to be console errors thrown at this line, maybe we can check if there is a `firePerimeterLayer` before fitting bounds. 
image

**Review by harper-wavra — CHANGES_REQUESTED** [review]
I know it hasn't passed linting yet but thought I would review now as I will be out for a bit after today.

Working great! Just a couple comments, 

-The gage table seems to be a bit off. 
image
- a couple console.logs 

I also wonder if right before we merge this, we update the polygon endpoints & change the `nldiPolygonQuery` url back so everything is done in this one issue

**Comment by amedenblik** [comment]
I have addressed linting and your review. I also reverted `nldiPolygonQuery` back to `https://streamstats.usgs.gov/nldipolygonservices/nldi_poly_query`. Let's ping @Anders-Hopkins to rename `https://streamstats.usgs.gov/nldipolygonservices/nldi_poly_query_new` to `https://streamstats.usgs.gov/nldipolygonservices/nldi_poly_query` before we merge this PR.

**Comment by amedenblik** [comment]
@harper-wavra this branch is working again and ready to be reviewed

**Review by harper-wavra — CHANGES_REQUESTED** [review]
I tested each endpoint. Delineation and Query by fire perimeter both looked & worked well. Query by basin seems to be getting hung up though. I tried on the https://test.streamstats.usgs.gov/national/ site and it's working there, so I think it might be related to changes in this branch.

**Comment by amedenblik** [comment]
I pulled in the `dev` branch, which fixed the Query by Basin issues. Reminder- let's not merge until nldi_poly_query is renamed.

**Review by harper-wavra — APPROVED** [review]
Looks good!

**Pull request merged**
